docs/source/faq.rst

```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
     mvn clean install -PIT
 
 Then connect to Tomcat using a remote debugger on port 8000, same as when normally debugging Tomcat.
+
+Why I not see anything in Admin Queues and Topics.
+--------------------------------------------------
```

Comment:
This should be rephrased to be clearer English, e.g.:

Why am I not seeing anything in "Admin Queues and Topics"?